### DISCUSSION TITLE
Overview button behaviour

### DISCUSSION THREAD

**Discussion by MotoUKRider** [devemux86/cruiser#156] [Q&A] [answered]
When you use the Overview feature (by pressing the turn display panel), if you zoom into the route and look for something, the method to get back to the 3D zoomed-in navigation of your current location is to repeat the process 2 more times, eg:

1st: Gets you route overview
Move map, zoom into something interesting
2nd: Returns to overview
3rd: Resumes close-in navigation view

If I press the location/target button, I get the map back to my own location - but the zoom level remains.  I am obviously stationary at this time!  I'm just wondering if there's a "cleaner" way to get back to own location at the "stopped" zoom level without too much pressing - perhaps by changing the Overview button caption when it is *not* going to show you the Overview, ie: "Resume"

Previously, I've been ignoring all of the above by pausing the navigation, using the map and then resuming navigation - much easier, but I'm just unsure if the above process needs a tidy-up?

**Comment by devemux86** [comment]
@MotoUKRider thanks for the detailed report.

Indeed we could improve the overview function.
Currently it works with the steps you described.

Auto center will be done anyway after a few seconds: "Settings | Location | Advanced | Auto center map view".
And auto zoom will be done as well, if enabled after a few seconds.

What you mean is the 2nd press of overview, to resume **always** on close view of location?

**Reply by MotoUKRider** [reply]
Yes, I think that's what I'm asking!  The process now is toggle between Close-In/Overview, eg: A/B/A/B but if you add "C" (move+zoom activity) into the mix, you get A/B/C/B/A - so I perfectly understand *why* this works, just didn't think the "method" was obvious as to how you get back quickly.

**Comment by devemux86** [comment]
A more clear behavior is needed, so users can understand and get the expected result:
- **Overview button**: press for route overview
(every time it does route overview)
- **Location button**: press to return to location
(with 1 step)

**Reply by DolomitesTom** [reply]
What about the overview split into different zoom levels, for example 10%, 20%, 50%, 100% of the full overview on every finger tip (always from actual gps position)?

**Reply by devemux86** [reply]
@DolomitesTom this is also in my list, please see:

- https://github.com/devemux86/cruiser/discussions/94

**Reply by DolomitesTom** [reply]
Nothing is forgotten 😁

**Reply by devemux86** [reply]
> Nothing is forgotten 😁

I write all the ideas and go through them step by step! :slightly_smiling_face:

**Comment by devemux86** [comment, marked as answer]
Improved overview button is available in [Cruiser 3.0.20](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

- **Overview button**: press for route overview
(every time it does route overview)
- **Location button**: press to return to location

**Reply by DolomitesTom** [reply]
Works 🙂
